enterprise_gateway/enterprisegatewayapp.py

```diff
@@ -64,12 +64,21 @@ def remote_hosts_default(self):
     yarn_endpoint_env = 'EG_YARN_ENDPOINT'
     yarn_endpoint_default_value = 'http://localhost:8088/ws/v1/cluster'
     yarn_endpoint = Unicode(yarn_endpoint_default_value, config=True,
-        help="""The http url for accessing the Yarn Resource Manager. (EG_YARN_ENDPOINT env var)""")
+        help="""The http url for accessing the YARN Resource Manager. (EG_YARN_ENDPOINT env var)""")
 
     @default('yarn_endpoint')
     def yarn_endpoint_default(self):
         return os.getenv(self.yarn_endpoint_env, self.yarn_endpoint_default_value)
 
+    yarn_endpoint_security_enabled_env = 'EG_YARN_ENDPOINT_SECURITY_ENABLED'
+    yarn_endpoint_security_enabled_default_value = 'False'
+    yarn_endpoint_security_enabled = Unicode(yarn_endpoint_security_enabled_default_value, config=True,
```

Comment:
I think this needs to use the `Bool` traitlet, otherwise `yarn_endpoint_security_enabled` will reflect a string and comparisons against a boolean will not be correct.